test/e2e/specs/editor/various/block-bindings/custom-sources.spec.js

```diff
@@ -1069,7 +1069,9 @@ test.describe( 'Registered sources', () => {
 			await expect( initialButton ).toHaveText( 'Text Field Value' );
 			// Second block should be an empty paragraph block.
 			await expect( newEmptyButton ).toHaveText( '' );
-			await expect( newEmptyButton ).toBeEditable();
+			await expect(
+				newEmptyButton.getByRole( 'textbox' )
+			).toBeEditable();
```

Comment:
The Button block itself isn't content editable. The original check was incorrect.